src/rpdk/core/test.py

```diff
@@ -28,7 +28,7 @@
 LOG = logging.getLogger(__name__)
 
 DEFAULT_ENDPOINT = "http://127.0.0.1:3001"
-DEFAULT_FUNCTION = "TestEntrypoint"
+DEFAULT_FUNCTION = "TypeFunction"
```

Comment:
is there anything additional we need to do for this? We are sending something which conforms to [this](https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerTestPayload.java) and the "TypeFunction" entrypoint expects [this](https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/src/main/java/software/amazon/cloudformation/proxy/HandlerRequest.java). I think we may need to modify the constructed request to adhere.